librocksdb_sys/crocksdb/c.cc

```diff
@@ -3779,4 +3800,110 @@ int crocksdb_keyversions_type(const crocksdb_keyversions_t *kvs, int index) {
   return kvs->rep[index].type;
 }
 
+struct ExternalSstFileModifier {
+  ExternalSstFileModifier(Env *env, ColumnFamilyData *cfd, DBOptions &db_options)
+  :env_(env), cfd_(cfd), env_options_(db_options), table_reader_(nullptr) { }
```

Comment:
Can we extract env_options_ from env_?

Reply:
No, we can't. `EnvOptions` can only be initialized with `DBOptions`.